### ISSUE TITLE
TFT: Add option to disable touch

### ISSUE THREAD

**Issue by atanisoft** [stuartpittaway/diyBMSv4ESP32#6]
Using a TFT that does not have touch capability results in the TOUCH event being fired constantly. It would be good to have an option to disable this behavior. Or optionally ignore pressure=4095 to drop the event.

**Comment by stuartpittaway** [comment]
Ah, I'm not thinking of leaving the screen on all the time, so I was using touch to "wake" up the screen for a few minutes.

**Comment by atanisoft** [comment]
Perhaps have a timer to wake up in the case of touch events not working?

**Comment by stuartpittaway** [comment]
> Perhaps have a timer to wake up in the case of touch events not working?

I could use one of the I/O pins to wake it as well, so a simply push button would work.

**Comment by atanisoft** [comment]
> Can you try the code from the branch - https://github.com/stuartpittaway/diyBMSv4ESP32/tree/get-tft-to-do-something

Working on it, will post in a short bit.

> I could use one of the I/O pins to wake it as well, so a simply push button would work. Could be a use for J13

That's also a great option, where is J13?

**Comment by stuartpittaway** [comment]
> That's also a great option, where is J13?

Buried under the TFT screen, bottom left near the pin header

**Comment by atanisoft** [comment]
The "get-tft-to-do-something" branch looks pretty good on the controller. I still have the loopback from the test app connected on RX1/TX1 and the console shows failed to send (likely due to no response from modules). I'm not worried about that just yet though.

The display looks like it went to sleep after a few minutes of inactivity but before it was displaying a large "button" for "configure access point".

> Buried under the TFT screen, bottom left near the pin header

Looks like a great candidate for another 90deg 2pin connector with the wires going out to the right or up to the top between the relays.

**Comment by stuartpittaway** [comment]
Whilst I've got you.....

I'm working on the AVR programming code (it works!) but in that I want to disable tasks which may trigger whilst programming is in operation and affect the SPI bus, like SD card logging or touch.

I'm using a MUTEX on the SPI bus, which is fine and blocks the read/write, but I would prefer the task to be suspended so it didn't even try.

I suspended the tasks using 
```
vTaskSuspend(updatetftdisplay_task_handle);
```

but this still appears to trigger when a notification (from xTaskNotify) is triggered.

Any idea why or how I can stop the task completely, so it also ignores notifications?
